guide-cloud-deployment/neo4j-cloud-aws-ec2-ami/neo4j-cloud-aws-ec2-ami.adoc

```diff
@@ -119,17 +143,20 @@ If we're in a different region than us-east-1, we can run the following command
 [source,text]
 ----
 aws ec2 describe-images \
-  --filters "Name=product-code,Values=3yl6wt8a1ty4izxxfer68wkie" "Name=product-code.type,Values=marketplace" \
-  --query "Images[*].ImageId" \
-  --region [Region]
+  --filters "Name=name,Values=neo4j-enterprise-1-3.4.1*" \
+  --region us-west-2 --query "Images[*].ImageId"
```

Comment:
Do we want this to be `us-east-1` so it's consistent everywhere or does this explicitly need to be `us-west-2`?

Reply:
Given context (finding AMIs in other regions) functionally it doesn't matter, but for the sake of clarity and consistency I'm going to change it.